core/sql/sqlcomp/PrivMgr.cpp

```diff
@@ -894,3 +918,49 @@ void PrivMgr::setFlags()
   SQL_EXEC_SetParserFlagsForExSqlComp_Internal(INTERNAL_QUERY_FROM_EXEUTIL);
 }
 
+// ----------------------------------------------------------------------------
+// method::log
+//
+// sends a message to log4cxx implementation designed by SQL
+//
+// Input:
+//    filename - code file that is performing the request 
+//    message  - the message to log
+//    index    - index for logging that loops through a list
+//
+// Background
+//   Privilege manager code sets up a message and calls this log method
+//   This method calls SQLMXLoggingArea::logPrivMgrInfo described in 
+//      sqlmxevents/logmxevent_traf (.h & .cpp)
+//   logPrivMgInfo is a wrapper class around qmscommon/QRLogger (.h & .cpp)
+//      log method
+//   QRLogger generates a message calls the log method in 
+//      sqf/commonLogger/CommonLogger (.h & .cpp) 
+//   CommonLogger interfaces with the log4cxx code which eventually puts
+//      a message into a log file called ../sqf/logs/master_exec_0_pid.log.  
+//      A new master log is created for each new SQL process started.
+//
+// Sometimes it is amazing that things actually work with all these levels
+// of interfaces.  Perhaps we can skip a few levels...  
+// ----------------------------------------------------------------------------
+void PrivMgr::log(
+  const std::string filename,
+  const std::string message,
+  const int_32 index)
+{ 
+  // for now, don't log
+  return;
```

Comment:
I am curious about this. Why have we disabled logging? Is log4cpp (or is it log4cxx) our current logging framework? Is it working adequately?